### PR TITLE
Request tracing

### PR DESCRIPTION
**Pull request by mrm9084**
## Description
Adds a correlation-context to requests with RequestType and Host.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,3 +7,4 @@ script:
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
+dist: trusty
```
Comment (warrenzhu25) [inline .travis.yml]: What's the purpose of this change?

Comment (mrm9084) [reply .travis.yml]: I ran into this issue: https://travis-ci.com/microsoft/spring-cloud-azure/builds/120492482 with the build. I chatted with @sophiaso who found https://travis-ci.community/t/error-installing-oraclejdk8-expected-feature-release-number-in-range-of-9-to-14-but-got-8/3766/4 and https://travis-ci.community/t/oracle-jdk-11-and-10-are-pre-installed-not-the-openjdk-builds/785/16. It is thought that the distribution was selected randomly if not configured explicitly. https://docs.travis-ci.com/user/reference/trusty/.

File: spring-cloud-azure-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/ConfigHttpClient.java [hunk 2]
```diff
@@ -148,4 +172,38 @@ private static String copyInputStream(InputStream inputStream) throws IOExceptio
...
+     * 
+     * @param request The http request that will be traced, used to check operation being run.
+     * @return String of the value for the correlation-context header. 
+     */
+    private static String getTracingInfo(HttpUriRequest request) {
+        String track = System.getenv(RequestTracingConstants.AZURE_APP_CONFIGURATION_TRACING_DISABLED.toString());
+        if (track != null && track.equalsIgnoreCase("false")) {
+            return "";
+        }
+        String requestTypeValue = request.getURI().getPath().startsWith("/kv") ? RequestType.STARTUP.toString()
```
Comment (sophiaso) [inline spring-cloud-azure-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/ConfigHttpClient.java]: nit: for several adjacent lines of code, how about adding empty lines between lines to split the blocks for better readability.
